components/mail/sendmail/Makefile

```diff
@@ -182,22 +190,24 @@ COMPONENT_TEST_TRANSFORMS += \
     '-e "/^XXX_CC_XXX\\$$/d" '
 
 $(INSTALL_32):	$(INSTALL_64)
+$(TEST_64):	$(TEST_32)
 
 # common targets
-build:		$(BUILD_32_and_64)
+BUILD_TARGET:		$(BUILD_32_and_64)
 
-install:	$(INSTALL_32_and_64)
+INSTALL_TARGET:	$(INSTALL_32_and_64)
 
-test:		$(TEST_32_and_64)
+TEST_TARGET:		$(TEST_32_and_64)
 
```

Comment:
You can replace these with a simple BUILD_BITS= 32_and_64 (or, if you prefer 64 bit apps 64_and_32).

Reply:
great idea. I've adapeted